### DISCUSSION TITLE
Having to re-add wireguard masquerade rule on reboot

### DISCUSSION THREAD

**Discussion by OfficialMuffin** [pivpn/pivpn#1705] [Support] [answered]
I have a basic wireguard setup alongside PiHole on a Raspberry Pi 4 8GB with just 4 clients. When rebooted, communication between wireguard clients work but no Internet connection. When pivpn -d is used upon reboot, it asks me to add the masquerade rule again. It works! When checking the added rule there are multiple duplicates of the same rule. I delete them and start again. After the 2nd reboot, it still doing the same thing. Not sure what's going on.

**Comment by coolapso** [comment]
https://github.com/pivpn/pivpn#feedback--support

Please follow the guidelines. 
Edit your post and add the data requested on the issue template: https://github.com/pivpn/pivpn/blob/master/.github/ISSUE_TEMPLATE/pivpn-report.md

**Reply by OfficialMuffin** [reply]
The .md file doesn't exist. Do I use the .yml template instead?

**Reply by coolapso** [reply]
I made some changes recently (after this post) ... if you go to issues > new issue it should now request the needed information instead of having to edit the template.

**Comment by OfficialMuffin** [comment, marked as answer]
Moved issue to #1713
